restapi/src/test/java/io/stargate/web/docsapi/service/query/search/resolver/filter/impl/InMemoryCandidatesFilterTest.java

```diff
@@ -199,7 +199,11 @@ public void fixedPath() {
           withQuery(
                   TABLE,
                   "SELECT key, leaf, text_value, dbl_value, bool_value, p0, p1, WRITETIME(leaf) FROM %s WHERE p0 = ? AND leaf = ? AND p1 = ? AND key = ? LIMIT ? ALLOW FILTERING",
-                  documentId)
+                  "field",
```

Comment:
why do we change the `documentApi` tests in the ticket that is fixing data store API? Shouldn't those changes be separated?

Reply:
These test were incorrect. They did not validate bound variables because the bugs in `ValidatingDataStore` did not make it possible. After fixing `ValidatingDataStore` I re-added bound variable validation to the tests.

`withQuery(...)` methods are meant to match both the query text and specific parameter values so that the test code could provide different result sets for different parameters.